src/main/groovy/com/calgaryscientific/gradle/VeracodeUploadFileTask.groovy

```diff
@@ -34,7 +34,8 @@ class VeracodeUploadFileTask extends VeracodeTask {
 
     VeracodeUploadFileTask() {
         description = "Uploads all files from 'build/to-upload' folder to Veracode based on the application id provided"
-        requiredArguments << 'appId' << "maxUploadAttempts${VeracodeTask.OPTIONAL}"
+        requiredArguments << 'appId'
+        optionalArguments << 'maxUploadAttempts'
```

Comment:
why do we need maxUploadAttempts?

Reply:
Same as before, part of the existing code base that this PR is not addressing.

Reply:
To answer the question, here it has a number of upload attempts when failing to upload the file to veracode.